aikau/src/main/resources/alfresco/layout/AlfSideBarContainer.js

```diff
@@ -407,42 +407,23 @@ define(["dojo/_base/declare",
             var width = (this.hiddenSidebarWidth) ? this.hiddenSidebarWidth : this.initialSidebarWidth;
             domStyle.set(this.sidebarNode, "width", width + "px");
 
-            // Add some right padding so that the resize bar doesn't appear over the content, this is
-            // particular important with regards to scrollbars that might be present (see AKU-514)
-            domStyle.set(this.sidebarNode, "padding-right", "10px");
             this.lastSidebarWidth = width;
             this.resizeHandler();
-            if (this.resizeHandlerButtonNode)
-            {
-               domClass.remove(this.resizeHandlerButtonNode, "pop-out");
-               domClass.add(this.resizeHandlerButtonNode, "pop-in");
-            }
+
+            domClass.add(this.sidebarNode, "alfresco-layout-AlfSideBarContainer__sidebar--open");
+            domClass.remove(this.sidebarNode, "alfresco-layout-AlfSideBarContainer__sidebar--closed");
```

Comment:
This should be a single class that's added/removed as necessary whenever the states are mutually exclusive. There's no real convention that I'm aware of for how to pick which one, but it makes sense that the defined state is the non-default one, so presumable that'd mean using `--closed`.

When using a boolean to determine the state-change, I find it tidiest to do this ...

```javascript
domClass[bool ? "add" : "remove"](node, className);
```

... but I accept that might be a bit too concise for some!